tests/unit_tests/data_steward/validation/participants/writers_test.py

```diff
@@ -23,16 +23,21 @@ def setUp(self):
         self.dataset = 'bar'
         self.site = 'rho'
 
+    @patch('validation.participants.writers.StorageClient')
     @patch('validation.participants.writers.gcs_utils.get_drc_bucket')
     @patch('validation.participants.writers.bq_utils.wait_on_jobs')
-    @patch('validation.participants.writers.gcs_utils.upload_object')
     @patch('validation.participants.writers.bq_utils.load_csv')
-    def test_write_to_result_table(self, mock_load_csv, mock_upload, mock_wait,
-                                   mock_bucket):
+    def test_write_to_result_table(self, mock_load_csv, mock_wait, mock_bucket,
+                                   mock_storage_client):
         # pre-conditions
         bucket_name = 'mock_bucket'
         mock_wait.return_value = []
         mock_bucket.return_value = bucket_name
+        mock_client_bucket = MagicMock()
+        mock_client_blob = MagicMock()
+        mock_storage_client.return_value = mock_storage_client
```

Comment:
I wonder if this can be set to something else or removed. The circular reference can cause problems.